src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Verify.java

```diff
@@ -87,26 +90,36 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
         while (row.hasNext()) {
           Entry<Key,Value> entry = row.next();
 
-          if (rowText == null)
+          if (rowText == null) {
             rowText = entry.getKey().getRow();
+          }
 
           long curr = Long.parseLong(entry.getKey().getColumnQualifier().toString());
 
-          if (curr - 1 != prev)
-            throw new Exception(
-                "Bad marker count " + entry.getKey() + " " + entry.getValue() + " " + prev);
+          if (curr - 1 != prev) {
+            log.error("Bad market count. Current row: {} {}, Previous row marker: {}",
+                entry.getKey(), entry.getValue(), prev);
+            break;
```

Comment:
Will breaking stop it from printing other problems in the test data?  

Should this set errorFound?

```suggestion
            errorFound=true;
            break;
```

Reply:
I forgot to add these. and the ones below but yes, they should. I will do more testing on the breaks, but it should match the same behavior it previously did there. As I mentioned in the description, we could forgo the breaks and then log everything if that is the desired behavior.